frontend/js/src/settings/missing-data/MultiTrackMBIDMappingModal.tsx

```diff
@@ -0,0 +1,519 @@
+              )
+            );
+          }
+        }
+        try {
+          const results = await Promise.allSettled(promises);
+          const successfulMappings: Array<{
+            recordingMSID: string;
+            track: MatchingTracksResult;
+            response: PromiseFulfilledResult<{ status: string }>;
+          }> = [];
+          const failedMappings: Array<{
+            recordingMSID: string;
+            track: MatchingTracksResult;
+            response: PromiseRejectedResult;
+          }> = [];
+          // Iterating with a forEach instead of for example using lodash groupBy
+          // so we can use the index, preserved by Promise.allSettled,
+          // to get the track name for display purposes
+          results.forEach((res, idx) => {
+            const recordingMSID = entries[idx][0];
+            const trackMetadata: MatchingTracksResult = entries[idx][1];
+            if (res.status === "fulfilled") {
+              successfulMappings.push({
+                recordingMSID,
+                track: trackMetadata,
+                response: res,
+              });
+            } else {
+              failedMappings.push({
+                recordingMSID,
+                track: trackMetadata,
+                response: res,
+              });
+            }
+          });
+          if (failedMappings.length) {
+            const failureList = (
+              <div>
+                <ul className="list-group list-group-item-danger list-group-item-text list-unstyled">
+                  {failedMappings.map((item) => (
+                    <li>{item.track.title}</li>
+                  ))}
+                </ul>
+                With the following errors:
+                <br />
+                {uniq(
+                  failedMappings.map((f) => f.response.reason.toString())
+                ).join(`\n`)}
+              </div>
+            );
+            toast.error(
+              <ToastMsg
+                title={`Failed to link ${failedMappings.length} track${
+                  failedMappings.length > 1 ? "s" : ""
+                }:`}
+                message={failureList}
+              />,
+              {
+                toastId: "failed-linked-track",
+                type: "error",
+              }
+            );
+          }
+
+          if (successfulMappings.length) {
+            const successList = (
+              <ul className="list-group list-group-item-success list-group-item-text">
+                {successfulMappings.map((item) => (
+                  <li>{item.track.title}</li>
+                ))}
+              </ul>
+            );
+            toast.success(
+              <ToastMsg
+                title={`You linked ${successfulMappings.length} track${
+                  failedMappings.length > 1 ? "s" : ""
+                }!`}
+                message={successList}
+              />,
+              {
+                toastId: "linked-track",
+              }
+            );
```

Comment:
Can we remove the bullet points in this toast message by add the following css?

```
ul {
  list-style-type: none;
  margin: 0;
  padding: 0;
}
```

Also, do we need the green background color in the toast?

Reply:
Probably don't want all the `ul` elements to have no list styling, but I thought the same as you and removed the bullet styling with a css class in 04681da3f